lib/controller.js

```diff
@@ -380,12 +380,24 @@ controller.provisionTessel = function(opts) {
     });
 };
 
-controller.deployScript = function(opts, push) {
```

Comment:
Wasn't this changed in your previous PR that improved deploy? Or was this just overlooked on that one? It doesn't actually change the functionality for `restart` right?